src/setfit/config.py

```diff
@@ -0,0 +1,20 @@
+from dataclasses import dataclass
+from typing import Any, Optional
+
+from transformers import PretrainedConfig
+
+
+@dataclass
+class SetFitModelConfig:
+    """
+    Config for SetFitModel.
+
+    Parameters:
```

Comment:
nit since we try to use `Args` in our docstrings:
```suggestion
    Args:
```